http-clients/netty-nio-client/src/main/java/software/amazon/awssdk/http/nio/netty/internal/utils/NettyClientLogger.java

```diff
@@ -85,33 +62,10 @@ public void debug(Channel channel, Supplier<String> msgSupplier, Throwable t) {
             return;
         }
 
-        Supplier<String> finalMessage = prependChannelInfo(true, msgSupplier, channel);
+        Supplier<String> finalMessage = prependChannelInfo(msgSupplier, channel);
         delegateLogger.debug(finalMessage.get(), t);
```

Comment:
Nit: Since we always immediately call `get()` here, it might be simpler to just return a `String`.